tool/compute-synthetic-coverage.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
         const synthetic = args.synthetic_set.pipe(csvparse({ relax: true, delimiter: '\t' }));
         synthetic.on('data', (line) => {
             const requoted = Array.from(requoteProgram(line[2])).join(' ');
-            syntheticPrograms[requoted] = (syntheticPrograms[requoted] || 0) + 1;
+            const requotedNoType = requoted.replace(/(param:[a-zA-Z_]+):\S+/g, '$1');
```

Comment:
We have a function to do this used by the evaluation code...

Reply:
And also, the regular expression is wrong because parameter names can have numbers.